### DISCUSSION TITLE
Add a note for each check mark

### DISCUSSION THREAD

**Discussion by islamarr** [iSoron/uhabits#1253] [Feature Requests] [answered]
It will be nice If we let the user add a short **note** after check the habit.
 
For example, If the habit is:
_" Did you read a book today? "_
Then when the user check mark that he has read, then he can see a box (or a bottom sheet) that let him optionally add a shot note like that : 
_"I have read 100 pages today, it is a great achievement ! "_

**Comment by iSoron** [comment, marked as answer]
Thanks for the feature request, @islamarr. This is a duplicate of #194. It has already been implemented and will become available in Loop 2.1.
